app/build.gradle

```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 27
+    defaultConfig {
+        applicationId "com.example.roshni.attendance"
```

Comment:
com.example is the default package name and is just a placeholder. Play Store restricts uploading apps with this package name. So I would suggest refactoring it.

Reply:
@shridhargoel can we have the package name 
com.cyberlabs.woc.attendence_manager
or
com.cyberlabs.woc.attendence.manager .

Reply:
@ujjwalagrawal17 Yeah, nice suggestion.